rotkehlchen/accounting/pot.py

```diff
@@ -165,8 +165,8 @@ def add_acquisition(
             location=location,
             timestamp=timestamp,
             asset=asset,
-            taxable_amount=amount,
-            free_amount=ZERO,
+            taxable_amount=amount if taxable is True else ZERO,
+            free_amount=amount if taxable is False else ZERO,
```

Comment:
This does not look right. You are putting the amount 2x here.  Unless I misunderstand something, we probably need a test for this, as something should have gone boom boom here.

Reply:
Hm why 2x? If `taxable is True`, `amount` will go to `taxable_amount` property and `ZERO` will go to `free_amount`. Otherwise the amount goes to `free_amount` and `ZERO` goes to `taxable_amount`. Total sum is always equal to `amount`, no?

Reply:
Ah lol I misread the conditional. Okay okay! 

So then please do it differently though. You are doing two if checks for the same variable every time it gets here. Do the checks at line 161:
```python
if taxable is True:
    taxable_amount = amount
    free_amount = ZERO
else:
    taxable_amount = ZERO
    free_amount = amount
```

And I would still like to see one test for taxableand one for not taxable